velox/connectors/hive/storage_adapters/hdfs/tests/HdfsFileSystemTest.cpp

```diff
@@ -0,0 +1,316 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include "velox/connectors/hive/storage_adapters/hdfs/HdfsFileSystem.h"
+#include <boost/format.hpp>
+#include <folly/portability/GMock.h>
+#include <hdfs/hdfs.h>
+#include <atomic>
+#include <random>
+#include "HdfsMiniCluster.h"
+#include "gtest/gtest.h"
+#include "velox/common/file/FileSystems.h"
+#include "velox/exec/tests/utils/TempFilePath.h"
```

Comment:
club all Velox headers together on top?

Reply:
Im not sure what you are suggesting. Do you want me to put all the "velox" dependencies above the other ones?

so like this:

#include "velox/common/file/FileSystems.h"
#include "velox/connectors/hive/storage_adapters/hdfs/HdfsFileSystem.h"
#include "velox/exec/tests/utils/TempFilePath.h"
#include <boost/format.hpp>
#include <folly/portability/GMock.h>
...

Reply:
Since you already made this change, this is fine. I noticed that the order of includes is inconsistent in a few other files as well. I see here (https://github.com/facebookincubator/velox/blob/main/CODING_STYLE.md#cpp-style) that clang-format is supposed to handle this, but it isn't currently.

Reply:
Thanks for the link. I will review that again to fix other discrepancies